src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -16,16 +16,11 @@ class ScalaSettings extends Settings.SettingGroup {
 
   /** Other settings.
    */
-  val dependencyfile = StringSetting("-dependencyfile", "file", "Set dependency tracking file.", ".scala_dependencies")
   val deprecation = BooleanSetting("-deprecation", "Emit warning and location for usages of deprecated APIs.")
   val migration = BooleanSetting("-migration", "Emit warning and location for migration issues from Scala 2.")
-  val encoding = StringSetting("-encoding", "encoding", "Specify character encoding used by source files.", Properties.sourceEncoding)
```

Comment:
Not even *allowing* `encoding` to be specified will break a lot of builds. It is commonly seen in sbt builds to explicitly specify `-encoding utf-8`.